CONTRIBUTING.rst

```diff
@@ -66,3 +130,47 @@ Last thing
 
   The title of your PR will become the commit message, please craft it
   with care.
+
+How to make a new release
+-------------------------
+
+If you want to hit the ground running and publish a new release on a freshly set-up machine, the following should suffice:
+
+.. code-block:: bash
+
+    # Grab the sources and install the dev tools
+    git clone https://github.com/C4ptainCrunch/ics.py.git && cd ics.py
+    pip install tox poetry bumpversion --user
+
+    # Make sure all the test run
+    tox && echo "Ready to make a new release" \
+        || echo "Please fix all the tests first"
+
+    # Bump the version and make a "0.8.0-dev -> 0.8.0 (release)" commit
+    bumpversion --verbose release
+    # Build the package
+    poetry build
+    # Ensure that the version numbers are consistent
+    tox --recreate
+    # Check changelog and amend if necessary
+    vi CHANGELOG.rst && git commit -i CHANGELOG.rst --amend
+    # Publish to GitHub
+    git push && git push --tags
+    # Publish to PyPi
+    poetry publish
+
+    # Bump the version again to start development of next version
+    bumpversion --verbose minor # 0.8.0 (release) -> 0.9.0-dev
+    # Start new changelog
+    vi CHANGELOG.rst && git commit -i CHANGELOG.rst --amend
+    # Publish to GitHub
+    git push && git push --tags
+
+Please note that bumpversion directly makes a commit with the new version if you don't
+pass ``--no-commit`` or ``--dry-run``,
+but that's no problem as you can easily amend any changes you want to make.
+Further things to check:
```

Comment:
I'm not sure about the "but that's no problem". You mean probably to amend an existing commit with `git commit --amend`, right? Hmn, I would avoid that information. A commit done by bumpversion should be not diluted with something else.

```suggestion
Note, that bumpversion directly makes a commit when bumping a new version. If you
want to avoid this, add the option ``--no-commit`` or ``--dry-run`` to the command.

Further things to check:
```

Reply:
Yeah, `git commit --amend` was what I was thinking about, as I thought all release-related information being collected in a single commit would be nice. If that's not what we want, we should probably also change the `vi CHANGELOG.rst && git commit -i CHANGELOG.rst --amend` bash code / steps above.